modules/monitoring-about-querying-metrics.adoc

```diff
@@ -1,11 +1,11 @@
 // Module included in the following assemblies:
 //
-// * monitoring/managing-metrics.adoc
+// * monitoring/querying-metrics.adoc
 // * virt/logging_events_monitoring/virt-prometheus-queries.adoc
 
 :_content-type: CONCEPT
-[id="querying-metrics_{context}"]
-= Querying metrics
+[id="about-querying-metrics_{context}"]
```

Comment:
I believe this ID should match the filename exactly. This practice makes it easier to change filenames, IDs, and xrefs in concert. 
```suggestion
[id="monitoring-about-querying-metrics_{context}"]
```

Reply:
None of the other context IDs for other module files in the monitoring fileset use the prefix "monitoring", so I'm going to leave this as-is